src/resources/js/components/ListPresentations.vue

```diff
@@ -15,6 +15,12 @@
                 <v-list-item-title v-html="presentation.title"></v-list-item-title>
               </v-list-item-content>
 
+              <v-list-item-action>
+                <v-btn icon @click.prevent="goToPresentationEdit(presentation.id)">
+                  <v-icon color="grey lighten-1">mdi-pencil-outline</v-icon>
+                </v-btn>
+              </v-list-item-action>
```

Comment:
you should verify if the user as the rights to edit the presentation before showing the button,
you get the `auth_user_role` on a presentation object that you could use to check if the user should see the button or not

Reply:
Changed in 4078b47